lib/server.js

```diff
@@ -192,27 +213,27 @@ function handleResponse(opts, req, resp, response) {
         ], opts.startTime);
 
         if (response.body) {
+
```

Comment:
Should we add these to hop-to-hop headers? For our purposes they are, since we transparently decode gzip encoding.